addons/website/views/snippets/snippets.xml

```diff
@@ -139,6 +139,7 @@
                     </t>
                     <t t-snippet="website.s_card" t-thumbnail="/website/static/src/img/snippets_thumbs/s_card.svg"/>
                     <t t-snippet="website.s_share" t-thumbnail="/website/static/src/img/snippets_thumbs/s_share.svg"/>
+                    <t t-snippet="website.s_social_media" t-thumbnail="/website/static/src/img/snippets_thumbs/s_facebook_page.svg"/> <!-- TODO replace by new thumbnail-->
```

Comment:
Is there someone working on this? We should get it ASAP as FP wants it merged for Tuesday

Reply:
PDE has done a task (quite a while ago now) for the designer team. I think they haven't started yet (I asked again on discord on 12/20 and they said they were busy but would get to it as soon as they could). But FP and PDE told me that we could merge like this and change the thumbnails afterwards.

Reply:
Ok, fair enough, we can indeed change it later. It's sad since I don't think this is a 2 days job but fine :+1: